include/dxc/Support/dxcapi.extval.h

```diff
@@ -0,0 +1,34 @@
+#include "dxc/Support/dxcapi.use.h"
+#include <string>
+
+class DxcDllExtValidationSupport : public dxc::DxcDllSupport {
+  // DxcDllExtValidationSupport manages the
+  // lifetime of dxcompiler.dll, while the member, m_DxilSupport,
+  // manages the lifetime of dxil.dll
+protected:
+  dxc::DxcDllSupport DxilSupport;
+
+  std::string DxilDllPath;
+
+  HRESULT InitializeInternal(LPCSTR dllName, LPCSTR fnName) override;
+
+public:
+  // override DxcDllSupport's implementation of InitializeInternal,
+  // adding the environment variable value check for a path to a dxil.dll
```

Comment:
I think you meant for the comment to prefix the `InitializeInternal` method?
```suggestion
  // override DxcDllSupport's implementation of InitializeInternal,
  // adding the environment variable value check for a path to a dxil.dll
  HRESULT InitializeInternal(LPCSTR dllName, LPCSTR fnName) override;

public:
```